apps/console/package.json

```
@@ -57,7 +57,7 @@
         "@wso2is/react-components": "^2.1.13",
         "@wso2is/theme": "^2.0.71",
         "@wso2is/validation": "^2.0.3",
-        "@wso2is/features": "*",
+        "@wso2is/features": "2.14.10",
```

Let's define the version with the caret symbol
```suggestion
        "@wso2is/features": "^2.14.10",
```